src/v1/connectors/kube.js

```diff
@@ -247,7 +247,7 @@ export default class KubeConnector {
       spec: {
         scope: {
           name: resourceName,
-          resource: apiGroup ? `${kind}.${apiGroup}` : kind,
+          resource: apiGroup ? `${kind.toLowerCase()}.${version}.${apiGroup}` : `${kind.toLowerCase()}.${version}`,
```

Comment:
@ycao56 Was looking at these changes for a similar issue in search. I came across this PR (https://github.com/open-cluster-management/multicloud-operators-foundation/pull/368/files#diff-db969064287b232ae734b98ac660ae43d3be3faf11917af3ef1da3f8f5680253R68). It looks like you may need to remove the `version` from the second part of this conditional.
ex: 
`resource: apiGroup ? `${kind.toLowerCase()}.${version}.${apiGroup}` : `${kind.toLowerCase()}`,`

Reply:
@zlayne 

yu is on PTO, I will create a pr for this

Reply:
@zlayne Are you sure? That PR refers to `kind.apiGroup` being bad since it's missing the version, not `kind.version`.

Reply:
I could be mistaken, but what i got from the above doc line i posted was that you can use either `kind.version.group` or just `kind`. But not able to use `kind.version`. I would ask you to try and create a ManagedClusterView to get a pod - as pods apiVersion is `v1` so this code would use `resource: pod.v1` in the `ManagedClusterView`. (I tried yesterday and got an error - but was also having some env issues so could just be something on my end :) )

Reply:
Base here https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L33

I think our current code will not be parsed mistakenly, so nothing needs to change, in my opinion.

Reply:
After going through the whole logic flow in the source code 

I think we will visit this line first 

https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L31

then this line

https://github.com/open-cluster-management/multicloud-operators-foundation/blob/5a7decb805ccbc17e49817ce553f9b52bbdafb0c/pkg/utils/rest/mapper.go#L80

So remove `.version` or not seems doesn't matter as long as there are less than two `dot `in the sting

@dhaiducek @zlayne

Reply:
Shoot. I'm seeing the same thing creating a custom ManagedClusterView, @zlayne--thanks for testing that. Yeah, if `kind.version` is provided it can't find it at all...

Reply:
So in the source code, we will first visit this line

https://github.com/open-cluster-management/multicloud-operators-foundation/blob/5a7decb805ccbc17e49817ce553f9b52bbdafb0c/pkg/utils/rest/mapper.go#L75

then this line

https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L51

then this line

https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L51

if there is Kind.version format string, we will get Group = version here https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L82

Reply:
Thanks for confirming! Glad it wasnt an env error on my end :)